tests/E2E/OrganizationTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * This file is part of the contentful-management.php package.
+ *
+ * @copyright 2015-2017 Contentful GmbH
+ * @license   MIT
+ */
+
+namespace Contentful\Tests\E2E;
+
+use Contentful\Management\Query;
+use Contentful\Tests\End2EndTestCase;
+
+class OrganizationTest extends End2EndTestCase
+{
+    /**
+     * @vcr e2e_organization_get_collection.json
+     */
+    public function testGetOrganizations()
+    {
+        $organizations = $this->client->getOrganizations();
+
+        $this->assertCount(3, $organizations);
+
+        $organization = $organizations[2];
```

Comment:
This seems extremely fragile to me.
As there is no "single" endpoint, I think it might be worth to consolidate these 2 tests into a single one, and instead of testing actual properties, just test their existence/correct type.
As an alternative, we could do some manual filtering to get the correct Org from the array. The API seems to be ignoring filtering using `sys.id`, so, I think that's the only alternative...